lib/Backend/arm64/EncoderMD.cpp

```diff
@@ -843,22 +843,22 @@ EncoderMD::GenerateEncoding(IR::Instr* instr, BYTE *pc)
         break;
 
     case Js::OpCode::CMP_ASR31:
-        dst = instr->GetDst();
         src1 = instr->GetSrc1();
-        Assert(dst->IsRegOpnd());
+        src2 = instr->GetSrc2();
```

Comment:
dst should be null right? if so add an assert.

Reply:
assert added